presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java

```diff
@@ -569,6 +570,9 @@ private String getVarcharValue(Type type, Object value)
             if (type instanceof TinyintType || type instanceof SmallintType || type instanceof IntegerType || type instanceof BigintType) {
                 return ((Long) value).toString();
             }
+            if (type instanceof BooleanType) {
```

Comment:
What about calling this method (`getVarcharValue`) as `getPartitionStringValue` ?